x-pack/legacy/plugins/security/index.js

```diff
@@ -114,9 +114,6 @@ export const security = kibana =>
       const xpackInfo = server.plugins.xpack_main.info;
       securityPlugin.__legacyCompat.registerLegacyAPI({
         auditLogger: new AuditLogger(server, 'security', config, xpackInfo),
-        isSystemAPIRequest: server.plugins.kibana.systemApi.isSystemApiRequest.bind(
-          server.plugins.kibana.systemApi
-        ),
```

Comment:
The Security plugin no longer needs to use this API, since each `KibanaRequest` now contains an `isSystemRequest` attribute.